app/forms/admin/druid_search_form.rb

```diff
@@ -24,7 +24,7 @@ def normalize_druid
     def collection_or_work_present
       return if collection.present? || work.present?
 
-      errors.add(:druid, 'not found')
+      errors.add(:druid, 'druid not found')
```

Comment:
made this change to match with the new `user not found` message i added -- seems clearer to me